Library/ViewModels/RewardsCollectionViewModel.swift

```diff
@@ -90,16 +91,35 @@ public final class RewardsCollectionViewModel: RewardsCollectionViewModelType,
       }
       .map(first)
 
+    let goToManagePledge = Signal.combineLatest(
+      project,
+      selectedRewardFromId,
+      refTag
+    )
+    .filter { project, reward, _ -> Bool in
+      project.state == .live && userIsBacking(reward: reward, inProject: project)
+    }
+    .map { project, reward, refTag in
```

Comment:
We can't do this point free, correct?

Reply:
It didn't compile. Maybe because the point free syntax doesn't apply for constructors.

Reply:
Stupid Swift :trollface: